src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -170,7 +176,14 @@ public static void Initialize(Win32PlatformOptions options)
                 .Bind<IMountedVolumeInfoProvider>().ToConstant(new WindowsMountedVolumeInfoProvider())
                 .Bind<IPlatformLifetimeEventsImpl>().ToConstant(s_instance);
 
-            Win32GlManager.Initialize();
+            if (options.UseVulkan)
```

Comment:
Shouldn't we let user use opengl when vulkan initialization fails for some reason?

Reply:
Addressed